DepositeCalcTests/Tests/LoginPageTests.cs

```diff
@@ -36,7 +36,62 @@ public void ValidLoginTest()
             calculatorPage.WeitForReady();
 
             // Assert
-            Assert.True(calculatorPage.IsOpened());
+            Assert.True(calculatorPage.IsOpened(), "Incorrect credentials");
+        }
+
+        [TestCase("test.@test.com")]
+        [TestCase(".test@test.com")]
+        [TestCase("@test.com")]
+        [TestCase("(),:;<>[]@test.com")]
+        [TestCase(" ")]
+        [TestCase("")]
+        [TestCase("@test.com")]
+        [TestCase("test@test@test.com")]
+        [TestCase("TestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTest@test.com")]
+
+        public void InvalidEmailRemindPasswordTests(string userEmail)
```

Comment:
Please remove this line.
```suggestion
        [TestCase("TestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTestTest@test.com")]
        public void InvalidEmailRemindPasswordTests(string userEmail)
```